src/types/validators.go

```diff
@@ -74,6 +75,42 @@ func (p *Project) assignDefaultProcessValues() {
 	}
 }
 
+func (p *Project) validateProcessCommand() {
+	for name, proc := range p.Processes {
+		if proc.Entrypoint == nil {
+			if proc.Command == nil {
+				message := fmt.Sprintf("Error: Neither command nor entrypoint is set for process: %s", name)
+				fmt.Println(message)
+				log.Fatal().Msg(message)
+			}
+			proc.Entrypoint = &[]string{
+				p.ShellConfig.ShellCommand,
+				p.ShellConfig.ShellArgument,
+			}
+		} else {
+			if len(*proc.Entrypoint) == 0 && (proc.Command == nil || *proc.Command == "") {
+				message := fmt.Sprintf("If entrypoint is empty, command needs to be non-empty (procces: %s)", name)
+				fmt.Println(message)
+				log.Fatal().Msg(message)
+			}
+		}
+
+		if len(*proc.Entrypoint) == 0 {
+			proc.Executable = *proc.Command
+			proc.Args = []string{}
+		} else {
+			entrypoint := *proc.Entrypoint
+			proc.Executable = entrypoint[0]
+			proc.Args = entrypoint[1:]
```

Comment:
If `len(*proc.Entrypoint)==1` this code will crash.

Reply:
hmm, just tested it and it seems to work fine

Reply:
You are absolutely right!
`entrypoint[1:]` returns an empty slice if `len(*proc.Entrypoint)==1`